modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/system/TestSystemObjectDefinitionMetadata.java

```diff
@@ -35,7 +35,7 @@
 /**
  * @author Feliphe Marinho
  */
-@Component(immediate = true, service = SystemObjectDefinitionMetadata.class)
+@Component(service = SystemObjectDefinitionMetadata.class)
```

Comment:
@MrJohn1911 this looks very much wrong. It is not from your change, but since you are touching it. We should figure out what is going on here.
Test module should not have scr components, and also a scr component can only have default constructor. This class has only one non-default constructor, there is no way this thing can work as a component.

@tinatian please check with SF team to ban scr usage from test modules.